### PR TITLE
fix image and command line args

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

**Comment by vishh** [comment]
@mindprince

File: Makefile [hunk 1]
```diff
@@ -14,8 +14,8 @@
 GO := go
 DOCKER := docker
 TAG?=$(shell git rev-parse HEAD)
-REGISTRY?=gcr.io/google-containers
-IMAGE=node-termination-handler
+REGISTRY?=gcr.io/k8s-image-staging
+IMAGE=gke-node-termination-handler
```
Comment (mindprince) [inline Makefile]: Why are we adding gke-? Would this not work on GCE? Should we use gcp-?

File: deploy/k8s.yaml [hunk 2]
```diff
@@ -49,10 +49,10 @@ spec:
                 operator: Exists
       containers:
         # TODO: Update to an official image once the build is automated via GCB.
-      - image: gcr.io/vishnuk-cloud/node-termination-handler@sha256:66f68f10cb552cba66ebadd038c958d275e88b2dba59f242f6ab661fccfec0d7
+      - image: gcr.io/vishnuk-cloud/gke-node-termination-handler@sha256:d733806b403567478e8f4e772664c2f3896c6ac894c1f927c698513c26528448
         name: node-termination-handler
         command: ["./node-termination-handler"]
-        args: ["--logtostderr", "--exclude-pods=$(POD_NAME):$(POD_NAMESPACE)", "--kubeconfig=/var/lib/kubelet/kubeconfig", "-v=10"]
+        args: ["--logtostderr", "--exclude-pods=$(POD_NAME):$(POD_NAMESPACE)", "-v=10"] # "--kubeconfig=/var/lib/kubelet/kubeconfig"
```
Comment (mindprince) [inline deploy/k8s.yaml]: I personally don't like to commit commented out code. It's there in git history, so we can retrieve it later if we want.

But I wouldn't block on this.

**Comment by vishh** [comment]
Given the other operational mode (using kubelet's SA) isn't functional
anymore, I agree it makes sense to not keep those configs. Cleaned up the
config now.

File: Makefile (hunk 1 above)
Comment (mindprince) [reply Makefile]: Ping on this. LGTM otherwise.

Comment (vishh) [reply Makefile]: The images that are specific to gke stored in `k8s-image-staging` project seem to all have a `gke-` prefix. This is to differentiate between GCP specific and pure OSS k8s components.

**Pull request merged**